### DISCUSSION TITLE
Clarification of the unit of the metrics

### DISCUSSION THREAD

**Discussion by jakov-aspiretechnology** [galexrt/dellhw_exporter#65] [Q&A] [answered]
Hi,

We would appreciate if you could check and confirm whether we have taken the right units for the following Dell HW exporter related metrics:

dell_hw_chassis_power_fail_level - W
dell_hw_chassis_power_reading - W
dell_hw_chassis_power_warn_level - W
dell_hw_ps_rated_input_wattage - W
dell_hw_ps_rated_output_wattage -W
dell_hw_chassis_current_reading - Percent

Thanks so much in advance!

**Comment by galexrt** [comment, marked as answer]
I am unable to run `omreport` at the moment but from the tests for the `omreport` command parsing, those units should be correct.
You can check the `omreport chassis pwrmonitoring` output if it matches the metrics. This code is doing the parsing:

https://github.com/galexrt/dellhw_exporter/blob/b1fa7695ee9a3284baa42f9d8f5bfac7ef93f3a8/pkg/omreport/omreport.go#L393-L434

**Reply by jakov-aspiretechnology** [reply]
Many thanks for your response.
